.github/workflows/build_and_push_to_docker.yml

```diff
@@ -137,7 +136,7 @@ jobs:
         # selected R versions available for rocker/rstudio
         RVERSION: ${{ matrix.RVERSIONS }}
 
-  build-r-dev:
+  build-r-ubuntu:
```

Comment:
wait - why do we need `build-r` and `build-r-ubuntu`?

Reply:
because the naming changed; 3.6.3 and 4.0.0 are available with -ubuntu-18.04 and the others aren't.

if you have a better idea feel free to propose!

Reply:
it was that or add in another if clause; and i wasn't sure if we wanted to keep `rockerdev` or not; maybe we do.

Reply:
Oh ok I see, I assumed they did the os-specific versions for all versions of R. Or is it just that those exist only on ubuntu-18.04?

Reply:
yeah just ubuntu*

Reply:
https://hub.docker.com/r/rocker/verse/tags